src/resolvers/git.cr

```diff
@@ -351,10 +351,7 @@ module Shards
     end
 
     private def valid_repository?
-      File.each_line(File.join(local_path, "config")) do |line|
-        return true if line =~ /mirror\s*=\s*true/
-      end
-      false
+      capture("git config --get-regexp 'remote\\..+\\.mirror'").each_line.any?(&.==("true"))
     end
 
     private def origin_url
```

Comment:
Do you think we can do something similar to `origin_url` then to fix the problem related to https://github.com/crystal-lang/shards/issues/628?

Reply:
Yes, I think this should work similarly.